schema/deploy/shipping/views.sql

```diff
@@ -930,4 +930,59 @@ comment on view shipping.scan_return_results_v1 is
   'View of barcodes and presence/absence results for SCAN return of results on the UW Lab Med site';
 
 
+create or replace view shipping.reportable_hcov19_v1 as
+
+    select
+        presence_absence.presence_absence_id,
+        organism.lineage::text,
+        sample.identifier as sample,
+        sample_id.barcode as sample_barcode,
+        collection_id.barcode as collection_barcode,
```

Comment:
Izzy et al. need two identifiers for CLIA, but I believe the two identifiers they need are specifically:

1. The collection barcode (e.g. from `collections-scan`).
2. The extra compliance barcode (e.g. from `collections-clia-compliance`). This should be found in `sample.details->>'scan_barcode'` now, I think.

Both of these are in REDCap and must both be cross-checked. The sample barcode isn't assigned until the lab processes the sample, and that information is not in REDCap AFAIK...

I'll double-check on this with Izzy/Misja/Peter/etc.

Reply:
The SCAN REDCap project's reporting instrument suggests this as well.

Reply:
Asked in Slack: https://seattle-flu-study.slack.com/archives/GUUDWB9BN/p1584946163113500